Makefile

```diff
@@ -1,27 +1,30 @@
+PROFILE=release
+
 .PHONY: build release install uninstall clean test doc reindent
 
 build:
-	jbuilder build @install --dev
+	dune build @install
 
 release:
-	jbuilder build @install
+	dune build @install --profile=$(PROFILE)
```

Comment:
I think having just a single `build` rule should be enough now, as here: https://github.com/xapi-project/xen-api/blob/master/Makefile#L11-L12, and then you don't need to override default goal.
It would default to PROFILE=release, and one can override it to be PROFILE=dev